HW01/Ivan_Papushoi/ivan_papushoi_hw_01_t_03.rb

```diff
@@ -0,0 +1,16 @@
+STR_EXPRESSION = '2222 - 222 + 22 / 2 - 2'.freeze
+
+def print_evaluation
+  puts "#{evaluate_expression} = #{STR_EXPRESSION}"
+  print "It consists #{calculate_twos(STR_EXPRESSION)} twos"
+end
+
+def evaluate_expression
+  2222 - 222 + 22 / 2 - 2
+end
+
+def calculate_twos(equatation_text)
+  equatation_text.count '2'
```

Comment:
There's no need to send argument
```suggestion
  STR_EXPRESSION.count '2'
```

Reply:
Done